app/shared/express.js

```diff
@@ -0,0 +1,116 @@
+/* @flow */
```

Comment:
Is this used in client code? This should probably be in app/main or app/services atleast because we shouldn't be running web servers off of the renderer process